### PR TITLE
Comment on PRs after nightly/release

### PR DESCRIPTION
**Pull request by ashwindasr**
Comment on the merged PR(s) after a nightly/release is created.

### CODE REVIEW

File: pkg/jira/jira.go [hunk 1]
```diff
@@ -85,6 +89,26 @@ func (c *Verifier) ghUnlabeledPRs(extPR pr) ([]pr, error) {
 	return unlabeledPRs, nil
 }
 
+func (c *Verifier) commentOnPR(extPR pr, message string) (error, bool) {
+	// Get the comments from that PR
+	comments, err := c.ghIssueClient.ListIssueComments(extPR.org, extPR.repo, extPR.prNum)
```
Comment (hoxhaeris) [inline pkg/jira/jira.go:94]: Use the updated `ghClient` instead (see the first comment).

File: pkg/jira/jira.go [hunk 2]
```diff
@@ -85,6 +89,26 @@ func (c *Verifier) ghUnlabeledPRs(extPR pr) ([]pr, error) {
...
+		return err, false
+	}
+	// Check to see if the same message has already been posted.
+	for _, comment := range comments {
+		if strings.Contains(comment.Body, message) {
+			return nil, false
+		}
+	}
+	// If the message hasn't already been posted, post it.
+	err = c.ghCommentClient.CreateComment(extPR.org, extPR.repo, extPR.prNum, message)
```
Comment (hoxhaeris) [inline pkg/jira/jira.go:105]: Use the updated `ghClient` instead (see the first comment).

**Review by hoxhaeris — CHANGES_REQUESTED** [review]

File: pkg/jira/jira.go [hunk 3]
```diff
@@ -22,6 +22,10 @@ type Verifier struct {
 	jiraClient jira.Client
 	// ghClient is used to retrieve comments on a bug's PR
 	ghClient githubClient
+	// GitHub client to comment
+	ghCommentClient github.CommentClient
```
[the remark this answers is not in the document]
Comment (ashwindasr) [reply pkg/jira/jira.go]: Ahh, I'm a bit new to Go, so I wasn't sure if I could do that. Thank you! I've made the changes in https://github.com/openshift/release-controller/pull/515/commits/97e300f395f6bcf1c8e027c8b62ed7a632855ca8

File: pkg/jira/jira.go (hunk 2 above)
Comment (ashwindasr) [reply pkg/jira/jira.go]: Fixed in https://github.com/openshift/release-controller/pull/515/commits/97e300f395f6bcf1c8e027c8b62ed7a632855ca8

File: pkg/jira/jira.go [hunk 4]
```diff
@@ -101,6 +123,8 @@ func (c *Verifier) verifyExtPRs(issue *jiraBaseClient.Issue, extPRs []pr, errs [
 				errs = append(errs, newErr)
 				return false, "", errs, false
 			}
+			// Comment on the PR saying that this PR is included in the release
+			_, success = c.commentOnPR(extPR, message)
```
Comment (hoxhaeris) [inline pkg/jira/jira.go:127]: We should not override the `success` bool here, since it is used in a condition check later on.

**Review by hoxhaeris — APPROVED** [review]

**Comment by hoxhaeris** [comment]
\cc @bradmwilliams

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**